core-plugins/src/main/java/io/cdap/plugin/common/FileSetUtil.java

```diff
@@ -93,10 +85,6 @@ public static void configureORCFileSet(String configuredSchema, FileSetPropertie
    */
   public static void configureAvroFileSet(String configuredSchema, FileSetProperties.Builder properties) {
     properties
-      .setEnableExploreOnCreate(true)
-      .setSerDe("org.apache.hadoop.hive.serde2.avro.AvroSerDe")
-      .setExploreInputFormat("org.apache.hadoop.hive.ql.io.avro.AvroContainerInputFormat")
-      .setExploreOutputFormat("org.apache.hadoop.hive.ql.io.avro.AvroContainerOutputFormat")
       .setTableProperty("avro.schema.literal", configuredSchema)
```

Comment:
Is it needed? I mean this method is same as previous with this one additional property. Given whole "avro" thing was related to explore, I would assume it's not needed

Reply:
Noted. I will examine it more carefully in a separate PR handling hive related changes.